client/deposits/details/index.tsx

```diff
@@ -160,20 +166,64 @@ export const DepositOverview = ( {
 	);
 };
 
-export const DepositDetails = ( {
+interface DepositDetailsProps {
+	query: {
+		id: string;
+	};
+}
+
+export const DepositDetails: React.FC< DepositDetailsProps > = ( {
 	query: { id: depositId },
-}: {
-	query: { id: string };
-} ): JSX.Element => (
-	<Page>
-		<TestModeNotice topic={ topics.depositDetails } />
-		<ErrorBoundary>
-			<DepositOverview depositId={ depositId } />
-		</ErrorBoundary>
-		<ErrorBoundary>
-			<TransactionsList depositId={ depositId } />
-		</ErrorBoundary>
-	</Page>
-);
+} ) => {
+	const { deposit, isLoading } = useDeposit( depositId );
+
+	const isInstantDeposit = ! isLoading && deposit && ! deposit.automatic;
```

Comment:
> When `automatic` is false, it could mean instant / manual (not implemented yet).

💯 on server, we currently don't store the [`method` from the Stripe payout object](https://stripe.com/docs/api/payouts/object#payout_object-method), which explicitly states `instant` or `standard`. When we do implement manual "standard" deposits, using this field should do the trick.